plugins/org.eclipse.n4js/src/org/eclipse/n4js/internal/N4JSProjectSourceContainer.java

```diff
@@ -45,7 +45,8 @@ public IN4JSProject getProject() {
 	public SafeURI<?> getLocation() {
 		String location = getRelativeLocation();
 		if (!Strings.isEmpty(location)) {
-			return project.getLocation().appendPath(getRelativeLocation());
+			String linuxPath = location.replaceAll("\\\\", "/");
```

Comment:
Not sure it's always correct to replace backslash with slash under non-windows platforms.

So, assuming `#getRelativeLocation()` returns a file system path as a string, I'd do something like:
```java
String linuxPath = File.separatorChar != '/' ? location.replace(File.separator, "/") : location;
```
(note that I use `#replace()` instead of `#replaceAll()` to avoid the need for `Pattern#quote()`)

Ideally, if `File.separatorChar!='/'`, we would also have to escape existing forward slashes somehow before changing `File.separatorChar` to `'/'`.